api/proto/gcb_ci_provider.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package proto;
+
+option go_package = "github.com/spinnaker/kleat/api/client";
+
+import "permissions.proto";
+
```

Comment:
```suggestion
// Configuration for the Google Cloud Build Provider
```

Not sure how much value the top-level docs add, but just to be consistent with the rest of the protos?

Reply:
Oh, definitely this is useful, especially since we aren't (at least to date) documenting these fields in the `HalConfig` and `EchoConfig` protos, relying instead on this comment to document.